test/libsolidity/semanticTests/shifts/shift_negative_rvalue.sol

```diff
@@ -1,15 +1,15 @@
 contract C {
-    function f(int256 a, int256 b) public returns (int256) {
+    function f(int256 a, uint256 b) public returns (int256) {
         return a << b;
     }
 
-    function g(int256 a, int256 b) public returns (int256) {
+    function g(int256 a, uint256 b) public returns (int256) {
         return a >> b;
     }
 }
 
 // ====
 // compileViaYul: also
 // ----
-// f(int256,int256): 1, -1 -> FAILURE
-// g(int256,int256): 1, -1 -> FAILURE
+// f(int256,uint256): 1, -1 -> 0
+// g(int256,uint256): 1, -1 -> 0
```

Comment:
I think this file needs to be renamed since what happens now is an underflow.

Reply:
@leonardoalt Should we allow this operation at all? I think it should fail

Reply:
The code itself looks fine. I just wonder if the `-1` is then interpreted as `uint(-1)` which underflows but should work

Reply:
Ok, will rename test to express that underflow is what happens here.